src/nrncvode/cvodeobj.cpp

```diff
@@ -1050,6 +1050,11 @@ void NetCvode::set_CVRhsFn() {
     }
 }
 
+static Section* cv_rootsec(const Cvode* cv) {
+    NrnThread* nt = cv->nth_ ? cv->nth_ : nrn_threads;
+    return nt->_v_node[cv->ctd_[0].vnode_begin_index_]->sec;
```

Comment:
What's the difference between `vnode_begin_index_` and `rootnodecount_`. To me they sound different but the first seems to replace a use of the second here.

Reply:
Partial answer is that `rootnodecount_` no longer exists, but not sure why they'd mean the same thing.

Reply:
See cvodeobj.h below. rootnodecount_ and vnodecount_ got replaced by  begin and end indices for rootnodes and vnodes. Rootnodes are consecutive and immediately followed by vodes. So it is kind of an accident that the rootnodecount_ happens to be the vnodeindexbegin_ of ```cv->ctd_[0]```.